cli/test-utils/src/util.rs

```diff
@@ -689,6 +692,21 @@ pub fn dir_list<P: AsRef<Path>>(dir: P) -> Vec<String> {
         .collect()
 }
 
+/// Creates a cross-platform remapping string for use in tests
+///
+/// NOTE: This probably should be unnecessary, and remappings should probably
+/// be canonicalized.
+pub fn remapping_str(src: &str, dest: &str) -> String {
```

Comment:
I wonder if we should do that in ethers-solc instead - not sure if using / would break anything internally, but I think at a minimum it would be safe to use / in `Display`?

Reply:
that makes sense!